### PR TITLE
Status page UI(fixes #33)

### LINKED ISSUE #33: Status page design

### PR DESCRIPTION
**Pull request by BryanGazali**
fixes #33

Add Status Page UI

Screenshot_20220326-183652
Screenshot_20220326-183658

### CODE REVIEW

**Review by ComputerOnFire — APPROVED** [review]
looks good so far :+1:

File: lib/feature/status/widget/RPI_widget.dart [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                style: TextStyle(
+                  fontSize: 15.0,
+                  fontWeight: FontWeight.bold
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.fromLTRB(8.0, 0.0, 0.0, 0.0),
+                child: Image.asset(
+                  'assets/icons/ic_pencil.png',
+                  width: 30,
```
Comment (rrijal53) [inline lib/feature/status/widget/RPI_widget.dart:27]: change size to 24 and add padding

File: lib/feature/status/widget/network_widget.dart [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                  fontSize: 15.0,
+                  fontWeight: FontWeight.bold
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.fromLTRB(8.0, 0.0, 0.0, 0.0),
+                child: Image.asset(
+                  'assets/icons/ic_globe.svg',
+                  width: 30,
+                  height: 30,
```
Comment (rrijal53) [inline lib/feature/status/widget/network_widget.dart:28]: change size to 24

**Review by rrijal53 — COMMENTED** [review]
Add appbar and change the icons sizee

**Review by rrijal53 — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Pull request merged**